web/src/components/BottomActionSheet.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement, ReactNode } from 'react'
+import { BottomSheet, BottomSheetRef } from 'react-spring-bottom-sheet'
+import 'react-spring-bottom-sheet/dist/style.css'
+import styled from 'styled-components'
+
+import '../styles/BottomActionSheet.css'
+import { getSnapPoints } from '../utils/getSnapPoints'
+
+const ListContainer = styled.div`
+  padding: 0 30px;
+`
+
+const Title = styled.h1`
+  font-size: 1.25rem;
+  font-family: ${props => props.theme.fonts.web.contentFont};
+`
+
+type BottomActionSheetProps = {
+  title?: string
+  children: ReactNode
+}
+
+const BottomActionSheet = React.forwardRef(
```

Comment:
because the ref can not be passed properly and used in `PoisPage`
```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?
```